examples/analytics.amp.html

```diff
@@ -442,6 +442,18 @@
 </script>
 </amp-analytics>
 <!-- End SOASTA mPulse example -->
+
+<!-- Linkpulse example - please replace id with your Linkpulse id -->
+<amp-analytics type="linkpulse">
+  <script type="application/json">
+     {
```

Comment:
the indentation is weird. This is nitpicky but if you don't mind, can you fix it? 2 spaces for indentation.